google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -1013,14 +1211,25 @@ private void doneCallback(Throwable finalStatus) {
   }
 
   @GuardedBy("lock")
-  private AppendRequestAndResponse pollInflightRequestQueue() {
-    AppendRequestAndResponse requestWrapper = this.inflightRequestQueue.pollFirst();
+  private AppendRequestAndResponse pollInflightRequestQueue(boolean pollLast) {
+    AppendRequestAndResponse requestWrapper =
+        pollLast ? inflightRequestQueue.pollLast() : inflightRequestQueue.poll();
     --this.inflightRequests;
     this.inflightBytes -= requestWrapper.messageSize;
     this.inflightReduced.signal();
     return requestWrapper;
   }
 
+  @GuardedBy("lock")
+  private AppendRequestAndResponse pollLastInflightRequestQueue() {
+    return pollInflightRequestQueue(true);
```

Comment:
Please document the parameter (/*pollLast=*/true)

Reply:
Done